src/models/tiles/log/log-comment-event.ts

```diff
@@ -27,13 +27,15 @@ function processCommentEventParams(params: ILogComment, context: IContext) {
     const [_unit, facet, _investigation, _problem, section] = parseSectionPath(documentId) || [];
     const curriculumStore = facet === "guide" ?  context.teacherGuide : context.problem;
     const tileType = tileId && curriculumStore?.getSectionById(section)?.content?.getTileType(tileId);
-    return { curriculum: documentId, tileId, tileType, ...others };
+    const tileTitle = tileId && curriculumStore?.getSectionById(section)?.content?.getTileTitle(tileId) || "<no title>";
+    return { curriculum: documentId, tileId, tileTitle, tileType, ...others };
   }
 
   const document = documents.getDocument(documentId) || networkDocuments.getDocument(documentId);
   if (document) {
     const tileType = tileId ? document.content?.getTileType(tileId) : undefined;
-    return { document, tileId, tileType, ...others };
+    const tileTitle = document?.content?.getTileTitle(tileId) ?? "<no title>";
+    return { document, tileId, tileTitle, tileType, ...others };
   }
 
   console.warn("Warning: couldn't transform log comment event params for document:", documentId);
```

Comment:
Is it worth trying to include a tile title if we get to this point? If I understand correctly, if we reach this point, there won't be any tiles.

Reply:
Seems like it would be tough to find the tile's title without the document, so I think it makes sense to wait until someone complains before worrying about this.